src/config/emailContent.js

```diff
@@ -0,0 +1,27 @@
+const CONTACT_MESSAGE_FIELDS = {
+  first_name: 'First Name: ',
+  last_name: 'Last Name: ',
+  mobile_phone: 'Phone Number: ',
+  email: 'Email: ',
+  dob: 'Date of Birth: ',
+  requested_date: 'Requested Date: ',
+  requested_time: 'Requested Time: ',
+  appointment_type: 'Appointmet Type: ',
+  description: 'Description: ',
+  is_emergency: 'Is this an emergency? ',
+};
+
+export const generateEmailContent = (data) => {
+  const stringData = Object.entries(data).reduce((str, [key, val]) => {
+    return (str += `${CONTACT_MESSAGE_FIELDS[key]}: \n${val} \n\n`);
+  }, '');
+
+  const htmlData = Object.entries(data).reduce((str, [key, val]) => {
+    return (str += `<h3 class="form-heading" align="left">${CONTACT_MESSAGE_FIELDS[key]}</h3><p class="form-answer" align="left">${val}</p>`);
+  }, '');
+  return {
+    subject: `Appointment request`,
```

Comment:
I understand that right now, you would have kept the values for test, but it will be good, if we have more descriptive Subject like - 'Connectient Appointment Request', 'Connectient Appointment Request - <Provider Name>', 'Connectient Appointment Request | <Appointment Request Date> | <Provider Name>', etc. Something we should keep in mind while rolling this out in production.